man/systemd.service.xml

```diff
@@ -255,6 +255,31 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ExitType=</varname></term>
+
+        <listitem>
+          <para>Configures the process exit type for this service unit. One of <option>main</option> or
```

Comment:
Maybe "Specifies when the manager should consider the service to be finished" ("configures the process exit type" doesn't really say much). You can also drop "One of mian or cgroup", the bullet list below already contains this.

Reply:
makes sense

Reply:
Regarding dropping the enum value in the sentence, I followed the style of other parts of the docs, like with `Type=` or `TimeoutStartFailureMode`